src/components/05_pages/NodeForm/index.js

```diff
@@ -151,6 +163,29 @@ class NodeForm extends React.Component {
     schema.properties.data.properties.attributes.properties[fieldName] ||
     schema.properties.data.properties.relationships.properties[fieldName];
 
+  resolveRequiredFiles = () => {
+    const unavailableFields = ['nid', 'uuid', 'vid', 'path'];
+    const requiredFields = this.props.schema.properties.data.properties.attributes.required.filter(
+      field => !unavailableFields.includes(field),
+    );
+    return Object.entries(this.state.entity.data.attributes)
+      .filter(([fieldName]) => requiredFields.includes(fieldName))
+      .filter(([fieldName, value]) => {
+        if (
+          typeof value === 'object' &&
```

Comment:
Do we need array validations here? The current condition will be true when the value is an Array